megatron/arguments.py

```diff
@@ -269,12 +269,13 @@ def validate_args(args, defaults={}):
         args.ffn_hidden_size = 4 * args.hidden_size
```

Comment:
Due to this PR's change, it's necessary to switch the order between this if condition (line 268, 269) and the `if args.swiglu and args.ffn_hidden_size is None:` below, otherwise the `if args.swiglu and args.ffn_hidden_size is None:` will never be triggered.